### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Natanel-Shitrit**

### CODE REVIEW

**Comment by Jcuhfehl** [comment]
Semi related: I wouldn't call mpv player a "new" feature anymore

**Comment by Natanel-Shitrit** [comment]
> Semi related: I wouldn't call mpv player a "new" feature anymore

Updated readme and PR name :)

File: README.md [hunk 1]
```diff
@@ -27,18 +27,17 @@ I am developing this application in my spare time.
...
     - SSA/ASS has limited styling support see [this issue](https://github.com/google/ExoPlayer/issues/8435)
-- **NEW** MPV Player
+- MPV Player
   - Container formats: mkv, mov, mp4, avi
   - Video codecs: H.264, H.265, VP8, VP9, AV1
   - Audio codecs: Opus, FLAC, MP3, AAC, AC-3, E-AC-3, TrueHD, DTS, DTS-HD
   - Subtitle codecs: SRT, VTT, SSA/ASS, DVDSUB
   - Optionally force software decoding when hardware decoding has issues.
-
+- Picture in picture mode
```
Comment (jarnedemeulemeester) [inline README.md:36]:
```suggestion
- Picture-in-picture mode
```

File: README.md [hunk 2]
```diff
@@ -27,18 +27,17 @@ I am developing this application in my spare time.
     - Support provided by ExoPlayer FFmpeg extension
   - Subtitle codecs: SRT, VTT, SSA/ASS, PGSSUB
     - SSA/ASS has limited styling support see [this issue](https://github.com/google/ExoPlayer/issues/8435)
-- **NEW** MPV Player
+- MPV Player
```
Comment (jarnedemeulemeester) [inline README.md:30]:
```suggestion
- mpv
```

**Review by jarnedemeulemeester — CHANGES_REQUESTED** [review]
Good catch! The README definitely needs an update.

While you're correcting it I would also use the correct naming from now on.

**Review by jarnedemeulemeester — APPROVED** [review]

**Pull request merged**